scripts/build-desktop.sh

```diff
@@ -30,7 +30,7 @@ title "Bundling Desktop js Bundle Using Webpack"
 info " • ELECTRON_ENV: $ELECTRON_ENV"
 info " • ENV file: $ENV_FILE"
 info ""
-npx webpack --config config/webpack/webpack.desktop.js --env envFile=$ENV_FILE
+npx webpack --config config/webpack/webpack.desktop.ts --env file=$ENV_FILE
```

Comment:
NAB, but should we consider updating `ENV` everywhere here to `ENVIRONMENT`?

Reply:
It's up to you, let me know what you'd like to do. I didn't think of it

Reply:
I think we should update it